pupa/scrape/schemas/vote.py

```diff
@@ -7,7 +7,7 @@
     "properties": {
         'identifier': {"type": "string", "blank": True,},
         'motion_text': {"type": "string" },
-        'motion_classification': {"items": {"type": "string", "enum": common.VOTE_CLASSIFICATIONS},
+        'motion_classification': {"items": {"type": "string"},
```

Comment:
(I'd hate to see everyone spell stuff differently for kicks, but I guess it's hard to nail down all the terms)